src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -254,11 +254,9 @@ def ls_factor(stream_length, area, avg_slope, m):
         return ls
 
 
-def stream_length(geom, drb=False):
```

Comment:
Let me double check. I searched through the whole project and don't think I saw any other invocations of the `stream_length` method.

There's a `streams(geom, drb=False)` method with similar logic that's called elsewhere.